ddtrace/bootstrap/sitecustomize.py

```diff
@@ -91,6 +91,11 @@ def update_patched_modules():
     if asbool(os.getenv("DD_RUNTIME_METRICS_ENABLED")):
         RuntimeWorker.enable()
 
+    # instrumentation telemetry writer should be enabled/started after the global tracer and configs
+    # are initialized
+    if asbool(os.getenv("DD_INSTRUMENTATION_TELEMETRY_ENABLED")):
```

Comment:
We need to enable telemetry after the tracer has been configured (TelemetryWriter.enable calls app-started which reads tracer configurations) and before integrations have been patched. We could move this change to line 116 or refactor TelemetryWriter.enable so it does not call app-started

Reply:
Tying this back to @brettlangdon

Reply:
I think either is fine (that is: calling this before or after calling `patch_all`)

If we want the app-started to be able to send integrations we'll need to make that change in the writer. Instead of sending an empty list always, try to gather any existing integrations registered with the writer and include there.

Maybe this is a good change to many anyways, even if we always call app-started before `patch_all` just so at worse case we get existing behavior (two payloads), but we are setup for the best case where app-started can send the integrations instead of sending a second payload.

wdyt?

Reply:
Yeah, i think that sending everything we can in the app-started message is the better option.

On this topic, because the writer is a periodic service, it only sends messages 60s seconds after the application has started. This is not ideal because we might loose data for short lived services. 
Munir told me it was done to have faster app startups but I wonder if the sending frequency couldn't be slightly higher.